src/platform/api.py

```diff
@@ -1,17 +1,17 @@
 """module to interact with the platform API"""
 
-import os
 from urllib.parse import urljoin
 
 import diskcache as dc
 import requests
 from beartype import beartype
 from deeporigin import auth
 from deeporigin.config import get_value
+from deeporigin.utils.core import _ensure_do_folder
 from deeporigin.utils.network import _get_domain_name
 
 
-def _make_request(endpoint: str) -> dict:
+def _make_get_request(endpoint: str) -> dict:
```

Comment:
Minor: this specifically implements `get` (not also `post`, `delete`, etc.) requests. The previous name reflected this.

Reply:
@jonrkarr so the current name is better, right?

Reply:
You're right. I mixed up the previous/new.